dags/hivemind_etl_helpers/tests/integration/test_notion_extractor.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from unittest.mock import Mock
+
+from dags.hivemind_etl_helpers.src.db.notion.extractor import NotionExtractor
+
+
+class TestNotionExtractorLive(unittest.TestCase):
+    def setUp(self):
+        """
+        Setup for each test case using mocks.
+        """
+        self.extractor = Mock(spec=NotionExtractor)
```

Comment:
Please mock the `NotionPageReader` and not the whole `NotionExtractor` class. We need to basically test the logic of our program and mocking the `NotionPageReader` will help us to test our own logic without depending on Notion API.

Also, after mocking the `NotionPageReader` this test case would need to be placed in unit tests directory as we've removed any of its dependencies and we're testing just one class logic.